app/src/main/AndroidManifest.xml

```diff
@@ -65,8 +65,7 @@
             android:parentActivityName=".activities.SettingsActivity" />
         <activity
             android:name=".activities.FwAppInstallerActivity"
-            android:label="@string/title_activity_fw_app_insaller"
-            android:parentActivityName=".activities.ControlCenterv2">
```

Comment:
Removed the parent activity because we don't want the back button to re-open the main activity, we want it to finish the current activity